packages/plugins/image/src/index.ts

```diff
@@ -96,6 +120,7 @@ export type ImagePluginState = ObjectStateType<{
   >
   alt: OptionalStateType<StringStateType>
   maxWidth: OptionalStateType<NumberStateType>
+  caption: ChildStateType
```

Comment:
```suggestion
  caption: OptionalStateType<ChildStateType>
```

`caption` should also be allowed to be empty so that plugins without a caption can be read -> currently there is an error at http://localhost:9001/?path=/story/article-initial-state--editor